sdk/search/azure-search/out/azure/search/index/_credential.pyi

```diff
@@ -0,0 +1,7 @@
+from typing import Any
+
+class SearchApiKeyCredential:
```

Comment:
I believe we call the other API key credential types <service>KeyCredential.

Reply:
Text Analytics has `TextAnalyticsApiKeyCredential`

https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_credential.py

My intention was to be consistent with that but happy rename if there is a better name

Reply:
Are you suggesting `CognitiveSearchApiKeyCredential` then? TBH the package naming structure probably needs review as well. "Search" is my understanding but a confirmation would be good.

Reply:
Yeah, my understanding is that it is Search as well. And I believe that you are correct (we changed it to **Api**KeyCredential to distinguish from other types of keys). 

I am not 100% up to speed on the various "search" services and how we ended up naming them. It would be good to ping the teams channel to make sure we use the same name across languages (I don't see any reason for Python to be different in this case)

Reply:
@bryevdv Yes, please don't use `Cognitive` in the Search client library names, as the API guidelines aim for technology focus rather than brand focus in top-level names.